front-end/src/assets/stylesheets/_variables.scss

```diff
@@ -5,6 +5,8 @@ $font-primary-ui: "GT-Walsheim";
 $color-primary: rgba(74, 74, 74, 1);
 $color-white: rgba(255, 255, 255, 1);
 $color-active-link: rgba(60, 143, 222, 1);
+$bg-color: #ededed;
```

Comment:
what is this bg-color used in? If it is shared then it can be here, but otherwise it can be defined inside the component itself.
If it is a global bg color, it can be set directly on `body`